doc/nrf/ug_thread_certification.rst

```diff
@@ -63,16 +63,19 @@ Complete the following steps to prepare for the certification tests:
 
 #. Build the certification image.
 
-   The :ref:`ot_cli_sample` sample is used as a base, modified with the :file:`harness/overlay-cert.conf` overlay file.
+   The :ref:`ot_cli_sample` sample is used as a base, modified with the following overlay files:
+   * :file:`../common/overlay-no-logging.conf`
+   * :file:`../common/overlay-ble.conf`
 
    .. code-block::
 
          cd ncs/nrf/samples/openthread/cli/
-         west build -b nrf52840dk_nrf52840 -- -DOVERLAY_CONFIG=harness/overlay-cert.conf -DCONFIG_OPENTHREAD_LIBRARY_1_1=y
+         west build -b nrf52840dk_nrf52840 -- -DOVERLAY_CONFIG="../common/overlay-no-logging.conf;../common/overlay-ble.conf" -DCONFIG_OPENTHREAD_LIBRARY_1_1=y
```

Comment:
Curious, why is Bluetooth enabling required for Thread certification ?
That is not clear to me at all, and thus I doubt the users understands why bluetooth must be enabed.

Reply:
Those two things are Using the same radio peripheral and we want to be sure that cooperation between them is works properly.